apps/admin/backend/schema.sql

```diff
@@ -18,7 +18,8 @@ create table write_in_candidates (
 );
 
 create table write_ins (
-  id varchar(36) primary key,
+  sequence_id integer primary key autoincrement,
```

Comment:
I'm probably lacking some context here, but does this provides more reliable ordering than the existing `created_at` timestamp?

Reply:
Yeah, I looked at the `created_at` timestamp and found that it didn't have the granularity to distinguish between write-ins added in sequence. I think that it uses unix "seconds since" date storage, unless that was purely how it was formatting data in the console. I think SQLite has other ways of storing dates but it seemed cleanest just to have a different field.